src/subscript/csv_stack/csv_stack.py

```diff
@@ -1,7 +1,88 @@
+"""Tool to stack/pivot CSV files, exposed as command line client,
+ERT workflow and ERT forward model"""
 import sys
 import re
+import logging
 import argparse
-import pandas
+from typing import Pattern
+
+import pandas as pd
+
+from ert_shared.plugins.plugin_manager import hook_implementation
+from res.job_queue import ErtScript
+
+from subscript import getLogger
+
+logger = getLogger(__name__)
+
+__MAGIC_STDOUT__ = "-"
+__MAGIC_STDIN__ = "-"
+
+DESCRIPTION = """Stack columns in a CSV exported file.
+
+All columns in your data set with a colon ":" in it, will be split such that the
+string after the colon will become a column value instead of its own column.
+Thus all columns called WOPT:A-1, WOPT:A-2, WOPT:A-3 etc will be merged into one
+column called WOPT, and you will have a column name called "WELL" that contains
+A-1, A-2, or A-3 as values.
+
+If importing the output CSV into Spotfire, you may then view and filter WOPT and
+friends by wellname, instead of selecting individual columns."""
+
+# The following string is used for the ERT forward model:
+EXAMPLE = """
+Put this in your ERT config::
+
+  FORWARD_MODEL CSV_STACK(<CSVFILE>=stackme.csv, <OUTPUT>=stacked.csv, <OPTION>="--keepminimal")
+
+"""  # noqa
+
+CATEGORY = "utility.transformation"
+
+# The following string is used for the ERT workflow documentation, note
+# the very subtle difference in variable name.
+WORKFLOW_EXAMPLE = """
+Add a file named e.g. ``ert/bin/workflows/CSV_STACK_WELLS`` with the contents::
+
+  CSV_STACK \"<CASEDIR>/share/results/tables/unsmry--montly.csv\" "--split" well "--keepminimal"
```

Comment:
You are more familiar with ERT workflow syntax than me, but are the `\`'s necessary?

Also, should we allow `-s` as alias for `--split` and `-k` for `--keepminimal` in the parser to avoid the need of `"`?
Or do you prefer to be explicit? I'm not sure myself...

Reply:
I think you are right, the backslashes are probably a remnant. It is not for ERT, but for Python.  Will remove

Reply:
Added `-s`, but `-k` would be ambiguous. Keeping long options in the example for the sake of being explicit and as a user remainder to quote double dashes.